lib/Model.js

```diff
@@ -106,6 +106,11 @@ function Model(o) {
         this._treatErrorsAsValues = options._treatErrorsAsValues;
     }
 
+    this._allowEnumerableMetadata = options.allowEnumerableMetadata ||
```

Comment:
* Is there still a need for allowEnumerableMetadata? If we're going with the obfuscated path?
* Also, if we're still experimenting with "allowFromWhenceYouCame", should we keep it off the public options, and keep it _ private only.